### DISCUSSION TITLE
your package is great, I think you can  Developing  for replace dat.gui

### DISCUSSION THREAD

**Discussion by z2586300277** [vanjs-org/van#193] [Ideas]
hello, my friend , I am from china . today I use your Van.js, it is vary good , before I use the dat.gui  ,but dat.gui Style can't be changed, it's defect， And your Van.js can change The style, but Van-ui if have some like dat.gui with  some to change  Number , color component ,then it's perfect,

**Comment by Tao-VanJS** [comment]
Thanks for the feedback! Glad you like it!😁 You can contribute if you want to add more customizations to VanUI.
